Python/test/test_Lgm_Vector.py

```diff
@@ -274,7 +274,9 @@ def test_SphToCart(self):
         # test an input check
         self.assertRaises(ValueError, Lgm_Vector.SphToCart, [1]*2, [2]*3, [3]*2)
         # test putting in lists
-        ans_tst = Lgm_Vector.SphToCart(zip(*invals)[0], zip(*invals)[1], zip(*invals)[2] )
+        ans_tst = Lgm_Vector.SphToCart(list(zip(*invals))[0],
+                                       list(zip(*invals))[1],
+                                       list(zip(*invals))[2])
```

Comment:
This is hardly optimization of the century, but wouldn't it make sense to do list(zip(*invals)) once, store the value, and then index it three times?

Reply:
It would, although this is just a formatting change over the previous incarnation - I didn't change code here.
I can pull the zip out onto a previous line and then index the list in the call.

Reply:
Whichever way you think is easier to read, honestly.